thoth/python/python_artifact.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+# thoth-python
+# Copyright(C) 2019 Kevin Postlthwait
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Representation a python module and all the files within."""
+
+import shutil
+import logging
+import requests
+import tempfile
+import zipfile
+import tarfile
+import hashlib
+import os
+from elftools.elf.elffile import ELFFile
+from elftools.common.exceptions import ELFError
+from typing import Iterator, Tuple
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class PythonArtifact:
```

Comment:
Can it be just `Artifact`? The package itself is Python-specific so the `Python` prefix feels redundant.